src/Exception/BadRequestException.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\Yii\Runner\Http\Exception;
+
+use Exception;
+
+final class BadRequestException extends Exception
```

Comment:
How would emitter react on throwing this exception? Would it properly respond with HTTP code 400?

Reply:
Done. I had to remake everything :)